src/parser/MyPluginParser.js

```diff
@@ -1,15 +1,18 @@
-import {DefaultParser} from 'leto-modelizer-plugin-core';
+import { DefaultParser } from 'leto-modelizer-plugin-core';
 
+/**
+ * Template of plugin parser.
+ */
 class MyPluginParser extends DefaultParser {
-  isParsable(fileName) {
+  isParsable(fileInformation) {
     /*
-     * Implement this to indicate which fileName your provider can manage.
+     * Implement this to indicate which fileInformation your provider can manage.
      *
      * You just have to return a Boolean to say if your parser can manage the file or not.
      *
      * By default, this function return false only on null/undefined fileName.
```

Comment:
update `filename` to `fileInformation name` or something else.

Reply:
done